mxcube3/ui/serverIO.js

```diff
@@ -32,6 +37,10 @@ export default class ServerIO {
       }
     });
 
+    this.hwrSocket.on('beam_value_change', (data) => {
```

Comment:
`beam_value_change` is a bit confusing, I would prefer something like `beam_position_change`